static/js/around-map.js

```diff
@@ -20,10 +20,14 @@ worker.addEventListener('message', response => {
     el.classList.add('marker')
     el.id = poi._id
 
-    const popup = new mapboxgl.Popup({offset: [10, 0]})
-      .setHTML(`<h2>${poi.name}</h2>
-                <h3>Comments:</h3>
-                <p>${poi.comments}</p>`)
+    let popup = new mapboxgl.Popup({offset: [10, 0]})
+    if(poi.comments.length === 0 || poi.comments[0] == [""]) {
```

Comment:
Proposal:
```
let content = `<h2>${poi.name}</h2>`
if(poi.comments[0]) {
  content += `<h3>Comments:</h3><p>${poi.comments}</p>`
}
popup.setHTML(content)
```

The `if` might be to adapt, you seem to have met some tricky weirdness.
And if that's the case you should comment. Did you have issues with the API?
If you put a comment and we fix the API later, we'll be able to check it's still the case or clean the code.